crates/libtiny_tui/src/tui.rs

```diff
@@ -626,6 +643,55 @@ impl TUI {
         tab.set_cursor(cursor);
     }
 
+    /// Shows "mentions" tab at the beginning of the tab list
+    pub(crate) fn show_mentions_tab(&mut self) -> Option<usize> {
+        let idx = self.new_misc_tab(MiscTab::Mentions, Some(0));
+        self.add_client_msg(
+            "Any mentions to you will be listed here.",
+            &MsgTarget::Misc(MiscTab::Mentions),
+        );
+        idx
+    }
+
+    pub(crate) fn show_help_tab(&mut self, messages: &[String]) {
```

Comment:
It's worth documenting here that this won't create a new tab and won't update existing tab's contents if a help tab already exists.

Also that this function will automatically select the help tab (when created or otherwise).